geotrek/common/parsers.py

```diff
@@ -466,6 +469,26 @@ def filter_attachments(self, src, val):
             return []
         return [(subval.strip(), '', '') for subval in val.split(self.separator)]
 
+    def has_size_changed(self, url, attachment):
+        try:
+            parsed_url = urlparse(url)
+            if parsed_url.scheme == 'ftp':
+                directory = dirname(parsed_url.path)
+
+                ftp = FTP(parsed_url.hostname)
+                ftp.login(user=parsed_url.username, passwd=parsed_url.password)
+                ftp.cwd(directory)
+                size = ftp.size(parsed_url.path.split('/')[-1:][0])
+                return size == attachment.attachment_file.size
```

Comment:
This will return true is size is the same. The fonction name is "has changed" though